.github/workflows/github-actions.yml

```diff
@@ -19,7 +19,7 @@
             java-version: '11'
             distribution: 'adopt'
         - name: Build with Maven
-          run: mvn clean test -Dbrowser="${{matrix.browser}}"
+          run:  mvn clean test -Dtest="TestGoogle"#purchase -Dbrowser="${{matrix.browser}}"
```

Comment:
why do you need pass this parameter

Reply:
I have specified which test I want to run, I still have the previous exercise as test

Reply:
la idea es que corran todos y que no tengas que estar agregandolos aquí